packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -84,7 +84,13 @@ const Cell = (props) => {
   const theme = useContext(ThemeContext);
   const cellRef = React.useRef();
   const [isSelectableCell, setIsSelectableCell] = useState(false);
-  const selectable = column.isSelectable && isSelectableCell;
+  /**
+   *  The cell is not selectable if one of the following applies:
+   *  - there is no onCellSelect method passed,
```

Comment:
Is this called out in the docs anywhere (that onCellSelect must be defined to make a cell selectable)?

Reply:
There are following mentionings in the documentation:
1. [Here](https://engineering.cerner.com/terra-ui/components/cerner-terra-framework-docs/compact-interactive-list/examples/cell-content) it says that **a cell is selectable unless it contains an interactive element, in which case the interactive element default controls apply**.
2. Required properties [here](https://engineering.cerner.com/terra-ui/components/cerner-terra-framework-docs/compact-interactive-list/examples/cell-content#required-properties) mention that `onCall` method is needed in order to be called for cell selection event.
3. [isSelectable](https://engineering.cerner.com/terra-ui/components/cerner-terra-framework-docs/compact-interactive-list/about#column) column prop with the description **A boolean value indicating whether the cells in this column are selectable.** It needs to be set to false in order to make all the cells in that semantic column non-selectable.

Reply:
UPD: I just added mentioning that `isSelectable` prop set to false would render the cell non-selactable [in this commit](https://github.com/cerner/terra-framework/pull/1972/commits/53c7adae382eafe65474d04d5dead2f2a54d0089)

Reply:
It might be worth calling out explicitly that "if onCellSelect is not defined, the cell will not be selectable even if isSelectable is set to true" or something like that since we call out the other two cases explicitly [here](https://github.com/cerner/terra-framework/pull/1972/files#diff-3410c80582aa64e6b9828c900dee5e5faf01ec29da3b8eb8e98f3a10e1216a91R24), but I don't have strong feelings either way.